lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -608,7 +608,7 @@
 '''
 
 RETURN = '''
-ansible_docker_container:
+docker_container:
```

Comment:
Please document the reason for the change: https://github.com/ansible/ansible/issues/18461